io.jamulus.Jamulus.metainfo.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>io.jamulus.Jamulus</id>
+  <metadata_license>MIT</metadata_license>
+  <project_license>LicenseRef-proprietary=https://github.com/jamulussoftware/jamulus/blob/master/COPYING</project_license>
```

Comment:
This needs changing as explained by you in the comments.

Reply:
Fixed. Thanks!